hypothesis-python/docs/reference/strategies.rst

```diff
@@ -1,15 +1,216 @@
+.. _strategies:
+
+Strategies reference
+====================
+
+Primitives
+----------
+
+.. autofunction:: hypothesis.strategies.none
+.. autofunction:: hypothesis.strategies.nothing
+.. autofunction:: hypothesis.strategies.just
+.. autofunction:: hypothesis.strategies.booleans
+
+Numeric
+-------
+
+.. autofunction:: hypothesis.strategies.integers
+.. autofunction:: hypothesis.strategies.floats
+.. autofunction:: hypothesis.strategies.complex_numbers
+.. autofunction:: hypothesis.strategies.decimals
+.. autofunction:: hypothesis.strategies.fractions
+
+Text
```

Comment:
"Strings", since we're including `binary()` here (which I think is the right place)